media/css/custom.scss

```diff
@@ -0,0 +1,490 @@
+  border-collapse: separate;
+  border-width: 0 0 1px 1px;
+  font-size: 20px;
+}
+
+th {
+  font-size: 14px;
+  font-weight: bold;
+  position: relative;
+  text-align: center;
+  vertical-align: middle;
+}
+
+td:hover,
+th:hover {
+  background-position: 0 -81px;
+  color: $light-black;
+}
+
+.events {
+  position: relative;
+
+  ul {
+    background: $light-grayish-blue;
+    border: 1px solid $white;
+    @include border-radius(3px);
+    -webkit-box-shadow: 0 8px 8px $dark-gray;
+    box-shadow: 0 8px 8px $dark-gray;
+    color: $gray;
+    display: none;
+    font-size: 15px;
+    list-style: none;
+    padding: 15px;
+    position: absolute;
+    text-align: left;
+    width: 600px; // 200px, wow banner
+    z-index: 1000;
+  }
+
+  li {
+    padding-bottom: 5px;
+
+    span {
+      color: $gray;
+      display: block;
+      font-size: 12px;
+      text-align: justify;
+
+      &.title {
+        color: $light-black;
+        font-weight: bold;
+      }
+    }
+  }
+}
+
+// applicants help
+
+.table-help-inf {
+  border: 1px solid;
+  border-collapse: collapse;
+  font-size: 13px;
+  margin-top: 30px;
+  width: 572px;
+
+  th {
+    border: 1px solid;
+  }
+
+  td {
+    border: 1px solid $light-gray;
+    padding-left: 3px;
+  }
+}
+
+.table-help {
+  text-align: center;
+}
+
+// student council
+
+.i2style {
+  border: 0 solid $white;
+  -moz-box-shadow: 1px -12px 0 $moderate-blue;
+  -webkit-box-shadow: 1px -12px 0 $moderate-blue;
+  box-shadow: 1px -12px 0 $moderate-blue;
+  color: $white;
+  cursor: pointer;
+  font: bold 24px Tahoma, Geneva, sans-serif;
+  margin: 0 auto;
+  padding: 10px;
+  text-shadow: 0 -1px 1px $light-black;
+  width: 575px;
+
+  &:active {
+    cursor: pointer;
+    position: relative;
+    top: 2px;
+  }
+
+  ~ img {
+    margin-bottom: 35px;
+  }
+}
+
+.widget-item {
+  height: 180px;
+  opacity: 0.75;
+  padding-bottom: 0;
+  padding-left: 0;
+  padding-right: 0;
+  position: relative;
+  -moz-transition: opacity ease 0.5s;
+  -ms-transition: opacity ease 0.5s;
+  -o-transition: opacity ease 0.5s;
+  -webkit-transition: opacity ease 0.5s;
+  transition: opacity ease 0.5s;
+
+  &:hover {
+    opacity: 1;
+  }
+
+  a {
+    display: block;
+    font-size: 22px;
+    height: 100%;
+    position: relative;
+    text-decoration: none;
+    width: 100%;
+
+    > div {
+      bottom: 0;
+      left: 0;
+      padding: 16px 16px 20px;
+      position: absolute;
+      right: 0;
+    }
+  }
+
+  &.blue {
+    a {
+      > div {
+        background-color: $widget-link-highlight;
+        color: $white;
+      }
+    }
+  }
+
+  &.schedule {
+    background: url("../img/schedule.jpg") center center;
+    background-size: cover;
+  }
+
+  &.announcement {
+    background: url("../img/announcement.jpg") center center;
+    background-size: cover;
+  }
+}
+
+#calendar {
+  margin: 0 auto;
+  max-width: 900px;
+}
+
+.link-box-graduate {
+  a {
+    background: $sidebar-background;
+    @include border-radius(6px);
```

Comment:
Expected item on line 461 to appear before line 460. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets